src/code/models/simulation.coffee

```diff
@@ -7,92 +7,89 @@ scaleInput = (val, nodeIn, nodeOut) ->
-    _.each links, (link) =>
-      return unless link.relation.isDefined
-      sourceNode = link.sourceNode
-      inV = sourceNode.previousValue
-      return unless inV               # we simply ignore nodes with no previous value
-      outV = @previousValue or @initialValue
-
-      inV = scaleInput(inV, sourceNode, this)
-
-      # Here we set maxIn to zero because we want to substract inV when
-      # we are in a `(maxIn - inV)` formula for accumulators (decreases by)
-      # Need a better long-term solution for this.
-      nextValue = link.relation.evaluate(inV, outV, 0)
-      value += nextValue
-  else
-    _.each links, (link) =>
-      if not link.relation.isDefined
-        count--
-        return
-      sourceNode = link.sourceNode
-      inV = if sourceNode.previousValue? then sourceNode.previousValue else sourceNode.initialValue
-      inV = scaleInput(inV, sourceNode, this)
-      outV = @previousValue or @initialValue
-      nextValue = link.relation.evaluate(inV, outV, link.sourceNode.max, @max)
-      value += nextValue
-    if count >= 1
-      value = value / count
-    else
-      value = @previousValue or @initialValue       # we had no defined inbound links
+  # collectors aren't calculated in this phase, but they do capture initial/previous values
+  startValue = if @previousValue? then @previousValue else @initialValue
+  return startValue if @isAccumulator or count < 1
+
+  value = 0
+  inValues = []
+
+  _.each links, (link) =>
+    return unless link.relation.isDefined
+    sourceNode = link.sourceNode
+    inV = if sourceNode.previousValue? then sourceNode.previousValue else sourceNode.initialValue
+    inV = scaleInput(inV, sourceNode, this)
+    outV = startValue
+    inValues.push link.relation.evaluate(inV, outV, link.sourceNode.max, @max)
+
+  # for now, if any a node points to a collector, it should use the scaled product
+  # ultimately this should be a default which can be overridden by the user, in
+  # which case the setting would presumably become part of the node model
+  useScaledProduct = !!(_.find @outLinks(), (link) -> link.targetNode.isAccumulator)
+  value = combineInputs(inValues, useScaledProduct)
 
   # if we need to cap, do it at end of all calculations
   value = @filterFinalValue value
 
   value
 
 SetAccumulatorValueFunction = (nodeValues) ->
-  # get add, subtract and transfer links
-  links = @inLinks('accumulator').concat(@inLinks('transfer'))
+  # collectors only have accumulator and transfer links
+  links = @inLinks('accumulator').concat(@inLinks('transfer')).concat(@outLinks('transfer'))
   return unless links.length > 0
 
-  value = 0
+  linkScaleFactor = 100
+
+  deltaValue = 0
   for link in links
     {sourceNode, relation, transferNode} = link
     inV = nodeValues[sourceNode.key]
     switch relation.type
       when 'accumulator'
         # TODO: check if outV is really equal to inV here - I'm not clear what outV is in this case'
```

Comment:
1. After a fairly thorough search of the code, I don't think it matters, because it doesn't look like it's used anywhere. It's part of the `scope` that is passed to the evaluated formulas, but I didn't find any formulas that reference it.
2. That said, by analogy with its handling in `RangeIntegrationFunction`, I think it's reasonable to assume that the output expected is the previous output of the node, so `@previousValue` or `@initialValue`, which is now captured in the code as `startValue` (as in `RangeIntegrationFunction`).

That's my story and I'm sticking to it. :wink: I've added a commit to that effect.